src/Application.vala

```diff
@@ -44,6 +45,8 @@ public class ValaLint.Application : GLib.Application {
             "Always return a 0 (non-error) status code, even if lint errors are found." },
         { "generate-config", 'g', 0, OptionArg.NONE, ref generate_config_file,
             "Generate a sample configuration file with default values." },
+        { "fix", 'f', 0, OptionArg.NONE, ref auto_fix,
+            "Fix any auto-fixable mistakes." },
```

Comment:
Can we add an entry in the README.md as well on how to use the flag, similar to this cli help text?

Reply:
Good point. Added.